internal/saucecloud/cucumber.go

```diff
@@ -0,0 +1,135 @@
+package saucecloud
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/internal/cucumber"
+	"github.com/saucelabs/saucectl/internal/framework"
+	"github.com/saucelabs/saucectl/internal/msg"
+
+	"github.com/saucelabs/saucectl/internal/job"
+)
+
+// CucumberRunner represents the SauceLabs cloud implementation
+type CucumberRunner struct {
+	CloudRunner
+	Project cucumber.Project
+}
+
+// RunProject runs the defined tests on sauce cloud
+func (r *CucumberRunner) RunProject() (int, error) {
+	var deprecationMessage string
+	exitCode := 1
+
+	m, err := r.MetadataSearchStrategy.Find(context.Background(), r.MetadataService, cucumber.Kind, r.Project.Playwright.Version)
```

Comment:
You can't use `cucumber.Kind` here, since Test-Composer will fail your request due to unknown framework.

Would be great if you could add a pipeline test to verify cucumber + playwright (just override the runner-version with whatever latest bundle you have from sauce-playwright-runner that works with cucumber), that way these errors can be caught easily 💪

Reply:
sure.